app/models/card.rb

```diff
@@ -2,16 +2,21 @@ class Card < ActiveRecord::Base
 validates :original_text, :translated_text, presence: true, uniqueness: { case_sensitive: false }
 validates :review_date, presence: true
 validate :must_not_be_equal
+scope :expired, -> { where('review_date <= ?', DateTime.now) }
+scope :random, -> { offset(rand(count))}
   before_validation(on: :create) do
-    self.review_date = DateTime.now.days_since(3)
+    review_actualize
   end
-  
+
   def must_not_be_equal
     if self.original_text.downcase == self.translated_text.downcase
       self.errors.add( :original_text,:must_be_diff_w_transl)
     end
   end
 
+  def review_actualize
+    self.review_date = DateTime.now.days_since(3)
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.